mlir/test/Conversion/GPUToLLVMSPV/gpu-to-llvm-spv.mlir

```diff
@@ -370,10 +295,23 @@ gpu.module @shuffles_mismatch {
 
 // Cannot convert due to variable shuffle width
 
-gpu.module @shuffles_mismatch {
+gpu.module @shuffles_mismatch extents<#gpu.spatial_extents<reqdSubgroupSize=32>> {
   func.func @gpu_shuffles(%val: i32, %id: i32, %width: i32) {
     // expected-error@below {{failed to legalize operation 'gpu.shuffle' that was explicitly marked illegal}}
     %shuffleResult, %valid = gpu.shuffle idx %val, %id, %width : i32
     return
   }
 }
+
+// -----
+
+// Cannot convert due to no subgroup size specified
```

Comment:
Can we check for the error you're emitting above? (That's emitted, right)?

Reply:
unfortunately there are not errors emitted